src/main/webapp/i18n/de/course.json

```diff
@@ -86,6 +86,17 @@
                 "title": "Onlinekurs",
                 "description": "Aktiviere diese Checkbox für externe Onlinekurse (z.B. auf edX), damit Studierende diesen Artemis-Kurs (über die LTI Schnittstelle) nutzen können. Dies ermöglicht auch eine automatische Kontoerstellung, sofern dies vom Artemis Systemadministrator eingestellt ist."
             },
+            "onlineCourseConfiguration": {
+                "title": "Onlinekurs Konfiguration",
+                "ltiKey": "LTI Key",
+                "ltiSecret": "LTI Geheimnis",
+                "userPrefix": "Nutzer Präfix",
```

Comment:
In German these crashes should either be one word or connected with a -

Reply:
Thanks, connected them with a -